kalite/distributed/static/css/distributed/khan-lite.css

```diff
@@ -924,6 +924,7 @@ to items you want to have the icon appear with.
 div#modal {
     position: absolute; /* makes the div go into a position that’s absolute to the browser viewing area */
     margin: auto;
+    padding-bottom: 50px;
```

Comment:
I think this much padding is not appropriate in all cases:
![ss](https://cloud.githubusercontent.com/assets/8888020/6422760/12460cd8-be91-11e4-808c-98ba2c9d5611.png)